packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js

```diff
@@ -8,11 +8,12 @@ export default class NutanixNodeGeneralInfoCtrl {
     this.nutanixNodeService = NutanixNode;
     this.$rootScope = $rootScope;
     this.Alerter = Alerter;
+    this.TRAVAUX_LINK =
+      TRAVAUX_LINK[coreConfig.getUser()?.ovhSubsidiary] || TRAVAUX_LINK.DEFAULT;
```

Comment:
**nitpick:** Since the link must be updated for US Manager, you can use `coreConfig.getRegion()`
```suggestion
    this.TRAVAUX_LINK =
      TRAVAUX_LINK[coreConfig.getRegion()] || TRAVAUX_LINK.DEFAULT;
```